models/5-reviews.models.js

```diff
@@ -0,0 +1,22 @@
+const connection = require('../db/connection')
+const { forEach } = require('../db/data/test-data/categories')
```

Comment:
Not sure you need this require statement here? It doesn't appear to be being used in this file so may as well remove it